### PR TITLE
Let the 2nd colon blink too

### PR DESCRIPTION
**Pull request by timoschwarzer**

### CODE REVIEW

**Comment by pablopunk** [comment]
This was my first approach and I thought it looked very weird, both at the same time, don't you think? Is there any digital clock that does this?

**Comment by pablopunk** [comment]
We could also make it alternate:

Kapture 2020-01-18 at 21 41 13

BTW the code for that is:

```jsx
style={{ opacity: blink && colonOpacity === 1 ? 0 : 1 }}
```

**Comment by timoschwarzer** [comment]
@pablopunk That looks definitely better. I also know a blink scheme where the colons blink twice a second. Maybe that's worth a try? Though we'd have to double the tick rate for that I think.

**Comment by timoschwarzer** [comment]
@pablopunk nvm, I just made a separate interval for the colons.

**Comment by timoschwarzer** [comment]
Hmm, Now gives me a 404 when I want to see details for the failed build... :(

File: pages/index.tsx [hunk 1]
```diff
@@ -128,17 +128,21 @@ export default class extends React.Component<IProps, IState> {
...
   componentDidMount() {
     this.tick()
     setInterval(() => {
       this.tick()
     }, 1000)
+    
+    // Let colons blink twice a second
+    setInterval(() => {
+      this.setState({
+        lastTickHadColon: !lastTickHadColon
```
Comment (pablopunk) [inline pages/index.tsx:142]: Here's the issue, `lastTickHadColon` is not in scope. You have to get it from state first inside the `setInterval` callback

**Review by pablopunk — CHANGES_REQUESTED** [review]

**Comment by pablopunk** [comment]
> Hmm, Now gives me a 404 when I want to see details for the failed build... :(

it seems I'm the only one allowed to see logs. See my code review to see the issue

File: pages/index.tsx (hunk 1 above)
Comment (timoschwarzer) [reply pages/index.tsx:142]: That happens when using the GitHub editor :P

**Comment by pablopunk** [comment]
hmmm I'm stil not sure about this https://time-hxv2pny5h.now.sh/?blink&seconds

**Comment by timoschwarzer** [comment]
Maybe like this? Minimal change but looks better imo.

File: pages/index.tsx [hunk 2]
```diff
@@ -120,25 +120,28 @@ export default class extends React.Component<IProps, IState> {
       seconds: '',
       mouseInteraction:
         this.props.showLink == null ? false : this.props.showLink,
-      lastTickHadColon: true
+      lastTickHadColon: false
```
Comment (pablopunk) [inline pages/index.tsx:123]: what's the purpose of this? it will start without a colon but both colons will still blink at the same time

Comment (timoschwarzer) [reply pages/index.tsx:123]: This makes the colons appear when the seconds change. Before they disappeared when the seconds changed.

Comment (pablopunk) [reply pages/index.tsx:123]: ahhh that makes sense, ok I'll merge this one and think about making them alternate, thanks for the PR!

**Pull request merged**